Marlin/Marlin_main.cpp

```diff
@@ -6193,6 +6193,10 @@ inline void gcode_M907() {
       if (code_seen(axis_codes[i])) digipot_current(i, code_value());
     if (code_seen('B')) digipot_current(4, code_value());
     if (code_seen('S')) for (int i = 0; i <= 4; i++) digipot_current(i, code_value());
+    if (code_seen('X')) digipot_current(0, code_value());
+    if (code_seen('Y')) digipot_current(1, code_value());
+    if (code_seen('Z')) digipot_current(2, code_value());
+    if (code_seen('E')) digipot_current(3, code_value());
   #endif
```

Comment:
Whoa....sorry for wasting your time. that was probably my most trivial oversight of the last 15 years. The idea that it doesn’t work (which was actually the hardware's fault) apparently stuck so deep in my brain, that I saw a bug that wasn't there because I was sure there had to be a bug. Thanks.